packages/app/src/specs/InlineSpecListHeader.vue

```diff
@@ -104,9 +105,20 @@ const emit = defineEmits<{
   (e: 'newSpec'): void
 }>()
 
-const inputFocused = ref(false)
+const runnerUiStore = useRunnerUiStore()
+
 const input = ref<HTMLInputElement>()
 
+watchEffect(() => {
+  if (runnerUiStore.isSpecsListInputFocused) {
+    // Wrapping in timeout since the inline spec tree auto-focuses
+    // the active spec after mount.
+    setTimeout(() => {
```

Comment:
Guessing you might have tried this but should we find the logic for "focus `activeSpec`", and do this same check there? Eg - something more reliable than just a `setTimeout` and hope that whatever milliseconds is enough.

Looks like that happens here https://github.com/cypress-io/cypress/blob/develop/packages/frontend-shared/src/composables/useVirtualListNavigation.ts#L10

So maybe we could pass a `shouldFocusItem` arg here? https://github.com/cypress-io/cypress/blob/develop/packages/app/src/specs/InlineSpecListTree.vue#L24

Then instead of doing the whole "focus the active spec then focus the input" we could just skip the active spec part, and we would not need the `setTimeout` work around.

Just an idea, not sure on the logistics or complexity of this, lmk what you think - can re-review/approve if this approach has some flaw.

Reply:
That's a good idea, will see if it's an easy add